### ISSUE TITLE
The license of the bundle is unclear

### ISSUE THREAD

**Issue by AkihiroSuda** [code-ready/crc#729]
I checked this repo, the installer archive, https://cloud.redhat.com/openshift/install/crc/installer-provisioned , and https://code-ready.github.io/crc/ , but could not find the license of the bundle.

**Comment by AkihiroSuda** [comment]
>  you accepted these when downloading the bundle.

Thanks for the answer, but is it shown only on the first download?

**Comment by gbraad** [comment]
We will make sure this information is available, but at the moment we are still in pre-GA. This did not get our attention as they these concern projects and products we consume.

**Comment by praveenkumar** [comment]
> Thanks for the answer, but is it shown only on the first download?

Yes terms and conditions are shown only first time when you login/register but if you check https://www.openshift.com/legal/terms/ have most of info around it but as @gbraad said we still need to put our attention around those concerns.

**Comment by gbraad** [comment]
Note: for RHEL CoreOS not a single license applies, but each individual component is under their respective license.

**Comment by gbraad** [comment]
Will ping legal again and PM

On Sun, May 10, 2020 at 4:54 PM stale[bot] <notifications@github.com> wrote:
>
> Closed #729.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.



-- 

   Gerard Braad | http://gbraad.nl
   [ Doing Open Source Matters ]
